recipe/meta.yaml

```diff
@@ -24,19 +27,20 @@ requirements:
     - setuptools_scm
     - wheel
     - cython >=0.29.24
-    - numpy {{ numpy }}
+    - numpy 2.0 # [py<313]
+    - numpy 2.1 # [py==313]
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - shapely >=1.7
-    - packaging >=20
-    - pyshp >=2.1
-    - matplotlib-base >=3.4
-    - pyproj >=3.1.0
+    - numpy >=1.23
```

Comment:
As we are building against numpy, do we need to at least set it as a run requirement?  Point of interest.